tests/docker-compose.yml

```diff
@@ -61,7 +61,7 @@ services:
         max-size: 100m
 
   data-broker:
-    image: dojot/data-broker:latest
+    image: dojot/data-broker:v0.4.2
```

Comment:
Shouldn't it be 0.4.3?

Reply:
Version v0.4.3 does not yet exist.
This is used only by automated integration tests.